### ISSUE TITLE
BatchNormalization

### ISSUE THREAD

**Issue by SnowdenLee** [Gsunshine/Enjoy-Hamburger#3]
Hello,

I have a question regarding the batch normalization. Is there a reason why you choose such a small momentum 3e-4 for the batch normalization?

Thank you in advance.

**Comment by Gsunshine** [comment]
Hi,

Thanks for your question! Regarding the BN momentum, we followed the hyperparameters in the previous work like EMANet. This is an old-school implementation, mainly because we finetune the ImageNet-pretrained backbone for the segmentation task with limited training samples and small batch size.  

According to my benchmark, it makes no difference for HamNet to adopt the classic 0.1 BN momentum under the ablation settings but shifts the peak validation performance to an earlier time. Maybe it can slightly improve the average performance or shorten the training schedule with the 0.1 BN momentum. This 0.1 BN momentum has already been applied to my collaborator Meng-hao Guo's new paper, [External Attention](https://github.com/MenghaoGuo/EANet).

Please let me know if you have additional questions. Thank you!
